pkg/operator/ceph/object/user/controller.go

```diff
@@ -18,202 +18,320 @@ limitations under the License.
 package objectuser
 
 import (
+	"context"
 	"fmt"
-	"reflect"
-	"time"
+	"strings"
 
-	"k8s.io/apimachinery/pkg/util/wait"
+	opcontroller "github.com/rook/rook/pkg/operator/ceph/controller"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"sigs.k8s.io/controller-runtime/pkg/source"
 
 	"github.com/coreos/pkg/capnslog"
 	"github.com/pkg/errors"
 	cephv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
 	"github.com/rook/rook/pkg/clusterd"
-	cephconfig "github.com/rook/rook/pkg/daemon/ceph/config"
 	"github.com/rook/rook/pkg/operator/ceph/object"
 	"github.com/rook/rook/pkg/operator/k8sutil"
+	corev1 "k8s.io/api/core/v1"
 	v1 "k8s.io/api/core/v1"
 	kerrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/client-go/tools/cache"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 const (
-	appName = object.AppName
+	appName        = object.AppName
+	controllerName = "ceph-object-store-user-controller"
 )
 
-var logger = capnslog.NewPackageLogger("github.com/rook/rook", "op-object")
+var logger = capnslog.NewPackageLogger("github.com/rook/rook", controllerName)
```

Comment:
nit: i'd prefer a shorter name to show up in the log file. What about `object-user`?

Reply:
I'd like to highlight it's a controller, I believe it makes sense, thoughts?

Reply:
All the logging in the same go package will have that same identifier, at least how we're using it today with the package `logger` var. So it really just helps quickly identify which go package we're in.